frontend/ui/widget/configdialog.lua

```diff
@@ -1104,20 +1108,31 @@ function ConfigDialog:onConfigMoreChoose(values, name, event, args, name_text, d
             else
                 -- CreDocument event handlers do their own refresh:
                 -- we can just redraw our frame
-                UIManager:setDirty(self, function()
-                    return "ui", self.dialog_frame.dimen
-                end)
+                if self.skip_paint then
+                    -- Redraw anything below the now hidden ConfigDialog
+                    UIManager:setDirty("all", function()
```

Comment:
Probably ought to be just ReaderUI instead of "all"? (IIRC, the other "all" instances in there are to catch ConfigDialog itself to repaint the panel. Here, it's hidden ;)).

(Unless there's potentially a stack of widgets in the way?).

Reply:
Especially given what `skip_paint` used to mean: it meant skip painting *ConfigDialog* ;).

Reply:
Yes, but there is a SpinWidget above ConfigDialog (which will get its own setDirty onShow).